app/formSchema/uiSchema/pages/submission.ts

```diff
@@ -1,14 +1,20 @@
 const submission = {
   'ui:order': [
     'submissionCompletedFor',
-    'submissionDate',
     'submissionCompletedBy',
     'submissionTitle',
+    'submissionDate',
   ],
   'ui:title': '',
   'ui:field': 'SubmissionField',
+  submissionCompletedFor: {
+    'ui:widget': 'ReadOnlySubmissionWidget',
+    'ui:options': {
+      'field-name': 'submissionCompletedFor',
```

Comment:
I made a custom read only widget for submission page because the design has different styling than the other read only widget (no padding, not bold). Rather than make a ton of `ui:options` flags it made sense to make one just for this page as it is the only place in the design that has it.

Reply:
If there's a different behaviour just for this page, then it makes sense to have a custom widget instead of using `readonly` (I'd like to know why our designs aren't consistent across pages though). 
I suppose that rendering the error within the widget makes sense in this case because we're not displaying other errors, just disabling the submit field (which is a separate rabbit hole we won't go into here...).

I still think having behaviour/ui options in the rjsf fields/widgets that is specific to one field is an antipattern and should be avoided. It's unclear when looking at the ui schema that this option will make the `ReadOnlySubmissionWidget` change its behaviour to something that is custom for this specific field. If one field needs a component to display something for that field only, let's make a widget for that field only